### DISCUSSION TITLE
Epic Marketplace Assets Integration

### DISCUSSION THREAD

**Discussion by Sekramo13** [Heroic-Games-Launcher/HeroicGamesLauncher#2420] [Q&A]
I'm trying to browse/download Unreal Engine/Epic Store Marketplace Assets.

The changelog for 2.0.0 Zoro Reads as follows:

[[General] Unreal marketplace is hidden by default now, there is a toggle in the settings to show it. (don't do that if you have a big selection of assets, right now Heroic freezes if you have 2000 assets or more, if this happens, edit the config file manually and set the option to false then restart Heroic.](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/CHANGELOG.md#:~:text=%5BGeneral%5D%20Unreal%20marketplace,then%20restart%20Heroic.)

Edit: Just adding the link to near the line here: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/CHANGELOG.md#whats-changed-10
Don't know why the above doesn't jump to the correct line.

I'm on version 2.6.1 now and I can neither find the setting in the Settings UI or in the ~/.config/heroic/config.json

I looked up the config.json file in code:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/3b05572e7e3c13b58a553aab7c95d91b0d54ad28/src/backend/game_config.ts#L235

Nothing seems to indicate that this setting exists....

My question:
How do I enabled the integration with Epic Store Marketplace Assets?

**Comment by JJones780** [comment]
I'm a new user and on the first run of the AppImage I see a setting in ~/.config/heroic/config.json  for version 2.14.1 that says:
`"showUnrealMarket": false,`

However, shutting down, changing to true, restarting and refreshing library does not show any new ( ie non-game)  items.

Edit: Looks like support was dropped in the fall of 2022?
This leads to the question: Was the config setting accidentally left in or is the feature coming back?
